spec/requests/api/v1/posters_request_spec.rb

```diff
@@ -118,5 +118,25 @@
     expect(poster[:data][:attributes][:vintage]).to eq(created_poster[:vintage])
     expect(poster[:data][:attributes][:img_url]).to eq(created_poster[:img_url])
   end
+
+  it "updates a poster" do
+    poster = Poster.create(
+      name: "REGRET",
+      description: "Hard work rarely pays off.",
+      price: 89.00,
+      year: 2018,
+      vintage: true,
+      img_url:  "https://plus.unsplash.com/premium_photo-1661293818249-fddbddf07a5d"
+    )
+    test_params = {name: "new_name"}
```

Comment:
were choosing to only update one param here, when we see a change to the posters, only the name KVP will be altered when this runs